stories/Patterns/FilterAndSearchBar/FilterAndSearchBar.scss

```diff
@@ -1,19 +1,17 @@
-@import '../../assets/scss/mixins';
-@import '../../assets/scss/breakpoints';
-@import '../../assets/scss/variables';
+@import "../../assets/scss/mixins";
+@import "../../assets/scss/breakpoints";
+@import "../../assets/scss/variables";
```

Comment:
@Ashish-srij Please use the proper linting as it should have single quotes

Reply:
@surbhi90 Done.